mkdocs.yml

```diff
@@ -23,6 +23,7 @@ nav:
   - 'how_to_guides/index.md'  # an index of clickable links to various guides
   - 'terminology/terminology.md'
   - 'roadmap/roadmap.md'  # actually the roadmap
+  - 'privacy_policy.md'
```

Comment:
attach a screenshot of how it looks through `mkdocs serve` please?

Reply:
![image](https://user-images.githubusercontent.com/62706609/225478813-374199c3-f377-4e72-8574-1f4b73957ec4.png)